src/utils.ts

```diff
@@ -0,0 +1,51 @@
+import { randomBytes } from "crypto";
+import { Account, Keypair, Operation, TransactionBuilder } from "stellar-base";
+
+/**
+ * Collection on util functions
+ *
+ * @namespace Utils
+ */
+export namespace Utils {
+  /**
+   * Returns a valid SEP0010 challenge which you can use for Stellar Web Authentication.
+   *
+   * @see [SEP0010: Stellar Web Authentication](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md)
+   * @param {Keypair} serverKeypair Server's keypair.
+   * @param {string} clientAccountID Client's Stellar account.
+   * @param {string} anchorName.
+   * @param {number} [timeout=300] Challenge duration (default to 5 minutes).
+   */
+  export function buildChallengeTx(
+    serverKeypair: Keypair,
+    clientAccountID: string,
+    anchorName: string,
+    timeout: number = 300,
+  ): string {
+    const account = new Account(serverKeypair.publicKey(), "-1");
+    const now = Date.now();
+
+    const transaction = new TransactionBuilder(account, {
+      fee: 100,
```

Comment:
I'll change this to use StellarBase.BASE_FEE

Reply:
I think you want to use fetchBaseFee and fetchTimebounds (in the Server class) instead of hardcoding values.

Reply:
@morleyzhi I check your suggestion, but I think for this particular scenario hardcoded values might be better for a couple of different reasons:

1. Both `fetchBaseFee` and `fetchTimebounds` add the overhead of an extra request to horizon to fetch data, which we don't need here (if the base fee is different from what the last ledger says, it doesn't matter)
2. Once we release stellar base with https://github.com/stellar/js-stellar-base/pull/209, we can use `StellarBase.BASE_FEE`
3. The idea behind `fetchTimebounds` is to help  if there is a time zone difference between the machine running this code and the Horizon's server -- but for this scenario, this code does not dependent on Horizon, but rather the server where the user wants to auth (which is the one running the code).